lib/keychain/sec.rb

```diff
@@ -149,11 +138,18 @@ def update_self_from_dictionary(cf_dict)
     # @return [Keychain::Keychain]
     def keychain
       out = FFI::MemoryPointer.new :pointer
-      status = Sec.SecKeychainItemCopyKeychain(self,out)
+      status = Sec.SecKeychainItemCopyKeychain(self, out)
       Sec.check_osstatus(status)
       CF::Base.new(out.read_pointer).release_on_gc
     end
 
+    # Removes the item from the associated keychain
+    def delete
```

Comment:
I think this only makes sense if some of the things that inherit from Sec::Base cease to do so (Access, TrustedApplication, ACL, maybe more), since those aren't (If I understand correctly) keychain items